common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -690,35 +811,35 @@ innovation_pandaren_ground = {
 	flag = global_maa
 }
 
-innovation_peacekeeper = {
+innovation_subjagators = {
 	group = culture_group_regional
 	culture_era = culture_era_early_medieval
-	icon = @peacekeeper
+	icon = @mogu
 
 	potential = {
-		this = culture:draenei
+		this = culture:mogu
 	}
 
-	unlock_maa = peacekeeper
+	unlock_maa = subjagators
 
 	flag = global_maa
 }
 
```

Comment:
Removed generic giants from mantid and nerubians.

Reply:
Do you want them to have generic giants?

Reply:
> Do you want them to have generic giants?

Remove*
It was a typo.